organizations/backends.py

```diff
@@ -35,9 +37,13 @@ def authenticate(self, *args, **kwargs):
         site = get_current_site()
         is_default_site = site.id == settings.SITE_ID
         if not is_default_site and is_site_configuration_enabled() and user and not user.is_superuser:
-            user_organizations = set(user.organizations.values_list('short_name', flat=True))
+            active_user_organizations = set(
+                models.UserOrganizationMapping.objects.active().filter(user=user).values_list(
+                    'organization__short_name', flat=True
+                )
+            )
```

Comment:
The code above is a little bit hard to reason about, please refactor for something that

```python
active_user_orgs = models.UserOrganizationMapping.objects.active().filter(
    user=user,
    # TODO: This may need to be refactored with the Tahoe `get_current_organization` helper and its sisters.
    organization__short_name=get_value('course_org_filter'),
)

if active_user_orgs.exists():
    return user
```